src/codeflare_sdk/cluster/cluster.py

```diff
@@ -875,6 +855,11 @@ def _map_to_ray_cluster(rc) -> Optional[RayCluster]:
                     protocol = "https"
             dashboard_url = f"{protocol}://{ingress.spec.rules[0].host}"
 
+    (
+        head_custom_resources,
```

Comment:
I am wondering about the use of `custom_resources` - I am not sure if it is clear here what will be expected from a user. To me custom resources are like RayClusters or LocalQueues. I understand that we want to avoid using just GPU as it is not broad enough. Could we use the term accelerators maybe?

Reply:
WDYT about `extended_resources` as defined here https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#consuming-extended-resources

Reply:
Yes I think that's better. We should also document this (when the notebooks are updated)  so that users know it is referring to accelerators. At a minimum with a comment beside the parameters in the notebook cluster config.